mexca/video/extraction.py

```diff
@@ -232,16 +192,12 @@ def extract(self, frame, boxes):
 
         return landmarks_np, aus
 
-    @staticmethod
-    def check_skip_frames(skip_frames):
-        if isinstance(skip_frames, int):
-            if skip_frames < 1:
-                raise ValueError('Argument "skip_frames" must be >= 1')
-        else:
-            raise TypeError('Argument "skip_frames" must be int')
 
     @staticmethod
-    def compute_centroids(embs, labels):
+    def compute_centroids(
+        embs: np.ndarray,
+        labels: List[Union[str, int]]
+    ) -> Tuple[List[float], Dict[str, int]]:
```

Comment:
Same here. this is an error.
```suggestion
    ) -> Tuple[List[float], Dict[str, float]]:
```